src/lib/core/ripple/ripple.md

```diff
@@ -139,3 +139,43 @@ const globalRippleConfig: RippleGlobalOptions = {
   terminateOnPointerUp: true
 };
 ```
+
+### Updating global options at runtime
+
+Developers are able to update the global `ripple` options at runtime by just injecting the
```

Comment:
I think this should be more prescriptive; i.e., "To change ripple options at run time, do this"

Reply:
Updated. I hope it's better now.